src/main/java/in/projecteka/consentmanager/consent/ConsentManager.java

```diff
@@ -40,13 +40,13 @@
 import java.security.PrivateKey;
 import java.security.Signature;
 import java.security.SignedObject;
-import java.util.ArrayList;
```

Comment:
I don't what is tab size and indent everyone use, I thought everyone use default. Isn't it true?